Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -361,6 +362,7 @@ namespace xr
 
         ~Impl()
         {
+            InputManager.reset();
```

Comment:
Can you add a comment to make explicit that this has to happen before `XrRegistry::Reset()` is called?